src/_data/styles/modern-stars.json

```
@@ -0,0 +1,6 @@
+{
+  "title": "Modern stars",
+  "author": "Alba Maldonado",
+  "stylesheet": "https://raw.githubusercontent.com/alba02/stylestage-modern-stars/080fc9be245267c2609ffa8683a179cd870e8e75/modern-stars",
+
```

Fix for the syntax here:
```suggestion
  "stylesheet": "https://raw.githubusercontent.com/alba02/stylestage-modern-stars/080fc9be245267c2609ffa8683a179cd870e8e75/modern-stars"
```